### PR TITLE
Enable VMWare inventory plugin

### PR DESCRIPTION
**Pull request by AlanCoding**
Connect https://github.com/ansible/awx/issues/3508

### CODE REVIEW

**Review by kdelee — APPROVED** [review]

**Review by ryanpetrello — APPROVED** [review]

File: awx/main/models/inventory.py
Comment (AlanCoding) [inline awx/main/models/inventory.py]: had this wrong, was given wrong number but not realizing it. Pushing next commit.

**Comment by AlanCoding** [comment]
this diff should be final, just waiting until the AM to merge

**Pull request merged**
